packages-exp/auth-compat-exp/demo/rollup.config.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import commonjs from 'rollup-plugin-commonjs';
+import json from 'rollup-plugin-json';
+import resolveModule from 'rollup-plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import typescriptPlugin from 'rollup-plugin-typescript2';
+import typescript from 'typescript';
+
+const plugins = [
+  sourcemaps(),
+  resolveModule(),
+  typescriptPlugin({
+    typescript,
+    compilerOptions: {
+      allowJs: true,
```

Comment:
what does this do?

Reply:
not sure, copied over from Firestore, probably safe to remove